code/README.md

```diff
@@ -15,3 +16,9 @@
   
   this file will using the data and request the time duration between each stop from mapbox direction API and than accumulates all 
   the time plus 5 min picking up or drop up time at each stop.
+  
+  # GetWayPoint.py
+  data: optimized route data
+  
+  this file will using the data and request the waypoints through the route along with the instructions for driveing
+
```

Comment:
Additionally, tell how you will run the file on the command line and what arguments are going to be added